worker/pkg/decompressor/decompressor.go

```diff
@@ -15,6 +21,12 @@ var (
 
 const limit = 1024 * 1024 * 1024 * 30 // 30GB
 
+const (
+	GCS_BUCKET_NAME               = "asset.cms.test.reearth.dev"
```

Comment:
Do not hard-code

Reply:
Ok, I wanted to keep the changes to minimum as this is a temporary solution.
Is it okay if i use the `.env` variables instead?